docs/manual/repository-management.md

```diff
@@ -0,0 +1,210 @@
+`pull-local` command can be used to migrate content from the "build"
+repository (in `bare-user` mode) into an `archive-z2` repository for
+serving to client systems.
+
+Following this section, we now have three repositories, let's call
+them `repo-build`, `repo-dev`, and `repo-prod`.  We've been pulling
+content from `repo-build` into `repo-dev` (which involves gzip
+compression among other things since it is a format change).
+
+When using `pull-local` to migrate content between two `archive-z2`
+repositories, the binary content is taken unmodified.  Let's go ahead
+and generate a new commit in our prod repository:
+
+```
+checksum=$(ostree --repo=repo-dev rev-parse exampleos/x86_64/stage-3-pass/standard`)
+ostree --repo=repo-prod pull-local repo-dev ${checksum}
+ostree --repo=repo-prod commit -b exampleos/x86_64/standard \
+       -s 'Release 1.2.3' --add-metadata-string=ostree.version=1.2.3 \
+	   --tree=ref=${checksum}
+```
+
+There are a few things going on here.  First, we found the latest
+commit checksum for the "stage-3 dev", and told `pull-local` to copy
+it, without using the branch name.  We do this because we don't want
+to expose the `exampleos/x86_64/stage-3-pass/standard` branch name in
+our "prod" repository.
+
+Next, we generate a new commit in prod that's referencing the exact
+binary content in dev.  If the "dev" and "prod" repositories are on
+the same Unix filesystem, (like git) OSTree will make use of hard
+links to avoid copying any content at all - making the process very
+fast.
+
+Another interesting thing to notice here is that we're adding an
+`ostree.version` metadata string to the commit.  This is an optional
+piece of metadata, but we are encouraging its use in the OSTree
+ecosystem of tools.  Commands like `ostree admin status` show it by
+default.
+
+## Derived data - static deltas and the summary file
+
+As discussed in [Formats](formats.md), the `archive-z2` repository we
+use for "prod" requires one HTTP fetch per client request by default.
+If we're only performing a release e.g. once a week, it's appropriate
+to use "static deltas" to speed up client updates.
+
+So once we've used the above command to pull content from `repo-dev`
+into `repo-prod`, let's generate a delta against the previous commit:
+
+```
+ostree --repo=repo-prod static-delta generate exampleos/x86_64/standard
+```
+
+We may also want to support client systems upgrading from *two*
+commits previous.
+
+```
+ostree --repo=repo-prod static-delta generate --from=exampleos/x86_64/standard^^ --to=exampleos/x86_64/standard
+```
+
+Generating a full permutation of deltas across all prior versions can
+get expensive, and there is some support in the OSTree core for static
+deltas which "recurse" to a parent.  This can help create a model
+where clients download a chain of deltas.  Support for this is not
+fully implemented yet however.
+
+Regardless of whether or not you choose to generate static deltas,
+you should update the summary file:
```

Comment:
Have we talked about summary files in one of the previous guides? I might have missed it.